CHANGELOG.md

```diff
@@ -1,16 +1,27 @@
 # Change Log
 
 ## v0.x.x Unreleased
+
+### New features
+
+### Maintenance and fixes
+
+### Deprecation
+
+### Documentation
+
+
+## v0.12.1 (2022 May 12)
 ### New features
 
 * Add `stat_focus` argument to `arviz.summary` ([1998](https://github.com/arviz-devs/arviz/pull/1998))
 
 ### Maintenance and fixes
 * `psislw` now smooths log-weights even when shape is lower than `1/3`([2011](https://github.com/arviz-devs/arviz/pull/2011))
```

Comment:
the just merged PRs seem to be missing from the changelog, can you try rebasing on top of main to make sure nothing will be lost in the automatic merge?

Reply:
Just rebased. The recent commits should be in there now.